packages/donor/src/components/basic/Popup/Popup.tsx

```diff
@@ -50,7 +52,10 @@ export default function Popup({
         {image && <img src={image} alt={"popup"} className={styles.image} />}
         <div className={styles.popupText}>
           <div className={styles.popupTextTitle}>{title}</div>
-          <div className={styles.popupTextContent}>{content}</div>
+          <div className={styles.popupTextContent}>
+            {content}
+            {children}
+          </div>
```

Comment:
you can remove the content prop if you use children

Reply:
Done